### PR TITLE
Mtbseq recipe

### PR DESCRIPTION
**Pull request by 000CU000**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/mtbseq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package:
+  name: mtbseq
+  version: "1.0.2"
```
Comment (johanneskoester) [inline recipes/mtbseq/meta.yaml]: Please move the version into a jinja2 variable and use it in the source url. This makes an update of the recipe easier.

Comment (000CU000) [reply recipes/mtbseq/meta.yaml]: Changed to jinja2 variable.

**Comment by johanneskoester** [comment]
This PR made me reinvestigate templating in conda-build 3. It turned out that PKG_VERSION is now a supported jinja variable that can be used in the meta.yaml. This makes using self-defined jinja2 variables superfluous. I took the liberty to adapt your PR accordingly.

**Comment by 000CU000** [comment]
Thank you for the changes. Still learning.

**Comment by johanneskoester** [comment]
All fine, the variable handling is actually in flow here at bioconda. Hence, there was not even a chance for you to do it right ;-).

**Pull request merged**